set.mm

```diff
@@ -108,6 +108,7 @@ been continuously enriched since then (list of contributors below). $)
 JY  Jonathan Yan
 FZ  Fan Zheng
 KZ  Kunhao Zheng
+TM  T M
```

Comment:
Since entries are sorted alphabetically by (last name, first name), this goes between RFL (line 68) and JPM (line 69) (note "M" < "Madsen"), like with ML and KP